runtime/browser/demo/explorer.html

```diff
@@ -0,0 +1,28 @@
+<link rel="import" href="../../../strategy-explorer/strategy-explorer.html"></link>
+
+<style>
+  strategy-explorer {
+    display: none;
+    background-color: white;
+    width: 100%;
+  }
+</style>
+
+<script>
+  (function() {
+    const hotKey = 'e'; // + CTRL
+    let explorer = document.body.appendChild(document.createElement('strategy-explorer'));
+    let explorerVisible = false;
+    window.addEventListener('keydown', e => {
+      if (e.ctrlKey && (e.key == hotKey)) {
+        e.stopPropagation();
+        e.preventDefault();
+        explorerVisible = !explorerVisible;
+        explorer.style.display = explorerVisible ? 'block' : '';
```

Comment:
on my screen, if the demo-flow wasn't set to display:none when strategy explorer was showing, then strategy explorer was behaving weirdly. 
but let's leave it as is for now, and see how it works.